recipe/build.sh

```diff
@@ -69,7 +68,6 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DENABLE_DOXYGEN=OFF \
       -DENABLE_LOGGING=ON \
       -DCMAKE_C_FLAGS_RELEASE=${CFLAGS} \
```

Comment:
This should be removed as well.

Reply:
What is the reason for the removal of these two?

Reply:
https://github.com/conda-forge/netcdf4-feedstock/issues/61
https://github.com/conda-forge/libnetcdf-feedstock/pull/56